### ISSUE TITLE
"MakeChromeSSB" quit unexpectedly.

### ISSUE THREAD

**Issue by zeartist** [dmarmor/osx-chrome-ssb-gui#6]
I tried with 2.0.1, 2.0.0 and 1.0.0. I get this same error (with all 3) after running the app I dragged it to my Applications folder. This error pops up in addition to the usual security warning popup (app downloaded from ...) It seems to run in an infinite loop on this error, since the same error dialog pops up no matter how many times I click open. Any suggestions?

**Comment by dmarmor** [comment]
Hi, sorry for the slow reply. To be clear, you dragged the app to your Applications folder, right? The app cannot be run from the DMG volume.

If that's the case, can you let me know what version of OSX you're running and what version of Chrome? Thanks.

**Comment by zeartist** [comment]
Hi! Yea I copied the app to my Applications folder first, than ran it.

Versions:
OSX: 10.7.5
Chrome: 43.0.2357.124 (64-bit)  (latest)

Before that I had 41.something... But Chrome was complaining about not being up-to-date. 
DISCLAIMER: Probably an important detail, but I have auto-updates pretty much turned off on everything here. I'm a software dev myself, and one thing I dislike is when my env keeps changing without my being aware. I prefer to run in the occasional wall of issues by not updating things too much, but rather doing updates manually when needed. Perhaps that is the cause? Thanks!

**Comment by dmarmor** [comment]
I fear the problem may be that you're running 10.7.5. I developed and have only tested this on 10.10 and have no access to a 10.7 system to try it on.

I'm assuming you just downloaded the DMG release. If you have the Xcode command-line tools installed on your system, you might try downloading the repository and compiling from the makefile to see if that gets it to run on your system. But I have no idea if it'll work or not. The project relies on several command-line tools (and Python), any of which may have incompatible versions back in 10.7.

Sorry about that! I do have future development plans (outlined in the Readme) to switch the SSB-maker over to being a Chrome extension rather than an app, in which case I think it'd be much more likely to run properly on older systems.

**Comment by zeartist** [comment]
Thanks! Yea, precisely, I tried via the dmg. Ok I guess I should just then upgrade OSX then and try again. I'll have to do a quick review of what could break else here but I think it should be fairly transparent. So I'll try and do that when I get a chance and will try again your tool after. Thanks again, I appreciate the help and look forward to try it.

**Comment by zeartist** [comment]
COOL!  This morning I thought to simply build the app from source instead, since that's usually the problem with incompatibilities. I scratched my head a bit, since I'm quite a noob at Mac / xcode / building anything other than java apps :-). But turns out it was quite easy. I already had xcode command-line tools installed, and tried "make" in the package. It build all fine and ran fine too, I was able to create a simple Gmail SSB with custom icon and all. So I'll give your solution a real try now, thanks again!

**Comment by zeartist** [comment]
COOL! Today I thought to simply build the app from source instead, since that's usually the problem with incompatibilities. I scratched my head a bit, since I'm quite a noob at Mac / xcode / building anything other than java apps :-). But turns out it was quite easy. I already had xcode command-line tools installed, and tried "make" in the package. It build all fine and ran fine too, I was able to create a simple Gmail SSB with custom icon and all. So I'll give your solution a real try now, thanks again!
